compliance/collection/main.go

```diff
@@ -201,7 +231,18 @@ func main() {
 	ctx = metadata.AppendToOutgoingContext(ctx, "rox-compliance-nodename", getNode())
 
 	stoppedSig := concurrency.NewSignal()
-	go manageStream(ctx, cli, &stoppedSig)
+
+	client, config, err := initializeStream(ctx, cli)
+	if err != nil {
+		log.Fatalf("error initializing stream to sensor: %v", err)
+	}
+
+	go manageStream(ctx, client, config, &stoppedSig)
+
+	if features.RHCOSNodeScanning.Enabled() {
+		scanner := nodescanv2.FakeNodeScanner{} // FIXME: Replace with real scanner (ROX-12971)
```

Comment:
[nit] There's a different format of TODO comments that our CI understands.
```suggestion
		// TODO(ROX-12971): Replace with real scanner
		scanner := nodescanv2.FakeNodeScanner{}
```